plugins/queue/smtp_forward.js

```diff
@@ -52,8 +52,15 @@ exports.get_config = function (connection) {
     const plugin = this;
 
     if (!connection.transaction) return plugin.cfg.main;
-    if (!connection.transaction.rcpt_to[0]) return plugin.cfg.main;
-    const dom = connection.transaction.rcpt_to[0].host;
+
+    var dom;
+    if (plugin.cfg.main.domain_selector === 'mail_from') {
+        if (!connection.transaction.mail_from) return plugin.cfg.main;
+        dom = connection.transaction.mail_from.host;
+    } else {
```

Comment:
No hugging elses, stroustrup brace style please

Reply:
Fixed this as well.